walnut/converter/converter.go

```diff
@@ -152,17 +152,28 @@ func sanitizeCmdLineArgs (input []string, taskName string) (output []string) {
 
 // GenerateTaskTemplate takes as input an array of pointers to task.Class
 // and writes them to a AliECS friendly YAML file
-func GenerateTaskTemplate(extractedTasks []*task.Class, dumpFileName string) (err error) {
-	dir, _ := os.Getwd()
-	path := filepath.Join(dir, dumpFileName, "tasks")
-	os.MkdirAll(path, os.ModePerm)
+func GenerateTaskTemplate(extractedTasks []*task.Class, outputDir string) (err error) {
+	path := filepath.Join(outputDir, "tasks")
+	path, _ = homedir.Expand(path)
+	_ = os.MkdirAll(path, os.ModePerm)
 
 	for _, SingleTask := range extractedTasks {
 		YAMLData, err := yaml.Marshal(&SingleTask)
 		if err != nil {
 			return fmt.Errorf("marshal failed: %v", err)
 		}
-		fileName := filepath.Join(path, SingleTask.Identifier.Name + ".yaml")
+
+		fileName := filepath.Join(path, SingleTask.Identifier.Name+".yaml")
+		if _, err := os.Stat(fileName); !os.IsNotExist(err) {
+			prompt := promptui.Select{
+				Label: fmt.Sprintf("%s already exists, overwrite?", fileName),
+				Items: []string{"yes", "no"},
```

Comment:
I think for binary choices you need to use `promptui.Prompt` instead of `Select`, with `IsConfirm` set to `true`. Please check this. https://godoc.org/github.com/manifoldco/promptui#Prompt

Secondly, does this prompt also react to `y/n` keys or does it expect the user to type `yes` or `no`? If so, this is not what we're looking for, all binary prompts should allow `y/n` keys followed by `Enter`, and hitting `Enter` without typing anything should select the default option (either `Y` or `N`).

Reply:
`promptui.Select` does not take text input. It provides choices, that the user can navigate and select. The default selected value is `yes` and the prompt cannot be exited without choosing a value.

Reply:
In that case, it seems to me that `promptui` is a deal-breaker:
- anything other than a `y` provided to `promptui.Prompt` [results in an error](https://github.com/manifoldco/promptui/issues/81) (???)
- even if we were to catch the error, we would then have to check the value of `result` to see if it's blank (and fall back to a default yes).

Reply:
I chose to drop `promptui` in favor of [survey/v2](https://github.com/AlecAivazis/survey) (as you originally recommended).